### ISSUE TITLE
ONNXRuntime outputs are different from PyTorch outputs

### ISSUE THREAD

**Issue by dtch1997** [microsoft/onnxruntime#4589]
**Describe the bug**
After exporting a PyTorch model to ONNX and performing inference on the same data using `onnxruntime.InferenceSession`, the model output is different from the output of the original PyTorch model. 

**Urgency**
None

**System information**
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: 1.4.0

Minimal example is provided in Colab so I am uncertain of the hardware details. 

**To Reproduce**
Please refer to this [Colab notebook](https://colab.research.google.com/drive/1cThbqXLHfkdXzeTtD6fxC3YkVZRc9RC-) for a minimal example. Run the cells in sequence. 

**Expected behavior**
ONNX Runtime should produce the same outputs as PyTorch to a high degree of precision (around 1e-16 for a float32 model)..

**Comment by dtch1997** [comment]
I resolved this issue after looking at [this post](https://stackoverflow.com/questions/57423150/outputs-are-different-between-onnx-and-pytorch). 

The reason this happens is that some layers behave differently between training and inference, most notably Dropout and BatchNorm layers. By calling `model.eval()` on PyTorch model I was able to obtain the expected result.
